src/components/Search.js

```diff
@@ -0,0 +1,16 @@
+import React,  { useState } from 'react';
+
+let Search = () => {
+  const [ searchText, setSearchText ] = useState('');
+
+  return <div>
+    <input
+      value={searchText}
+      onChange={(ev) => setSearchText(ev.target.value)}
+      type="text"
+      label="What do yoy want to watch?"></input>
+    <button onClick={() => setSearchText('')}>Reset Search</button>
```

Comment:
FYI
Ideally,  it's better to define callbacks we pass to JSX as variables in useCallback(otherwise additional rerenders can occur since every new () => {} is a new reference and React thinks that new variable is passed and rerendres component), but in some cases when you pass callbacks to native elements like input or button, this is overwork. So it's okay in this case, but be aware

Reply:
Thanks, will keep in mind.